packages/ui/src/components/va-data-table/hooks/useCommonProps.ts

```diff
@@ -0,0 +1,11 @@
+import { PropType } from 'vue'
+
+import { DataTableItem } from '../types'
+
+export const currentPage = { currentPage: { type: Number as PropType<number | undefined> } }
+
+export const items = { items: { type: Array as PropType<DataTableItem[]>, default: () => [] as DataTableItem[] } }
```

Comment:
This kind of naming might mislead. Would be better to name props like others in the `VaDataTable` component. For example `useDataTableItems`.